src/_locales/es/messages.json

```diff
@@ -440,47 +440,47 @@
+        "message": "Cuando utiliza Privacy Badger se une a la <a href='https://www.eff.org/' target='_blank'>Electronic Frontier Foundation</a> y a millones de otros usuarios en la lucha por la privacidad. Somos una organización sin ánimo de lucro que lucha por sus derechos en línea. ¡Gracias por unirse a nosotros!",
         "description": "Part of the 'donate' section on the intro page."
     },
     "options_domain_search_tooltip": {
-        "message": "Para excluir dominios, preceda su término de búsqueda con \"-\". Por ejemplo, \".co -.com\" mostrará dominios .co y .co.uk pero no .com.",
+        "message": "Para excluir dominios, preceda el término de búsqueda con \"-\". Por ejemplo, \".co -.com\" muestra dominios .co y .co.uk, pero no .com.",
         "description": "Tooltip for an \"information\" icon next to the domain search input on the Tracking Domains options page tab."
     },
     "intro_disable_button": {
-        "message": "Si crees que Privacy Badger está rompiendo una página (un vídeo no se reproduce, por ejemplo), puedes hacer clic en el botón \"Deshabilitar\" para desactivar Privacy Badger para ese sitio.",
+        "message": "Si cree que Privacy Badger está rompiendo una página (un vídeo no se reproduce, por ejemplo), puede hacer clic en el botón \"Deshabilitar\" para desactivar Privacy Badger en ese sitio.",
         "description": "Intro page paragraph. The quoted string ('Disable') should match the verb used for the 'popup_disable_for_site' message."
     },
     "intro_link_policy": {
         "message": "Política de privacidad",
         "description": "Shown at the bottom of the intro page, links to the EFF software and technology projects privacy policy."
     },
     "options_domain_list_no_trackers": {
-        "message": "Privacy Badger no ha detectado algún <a target='_blank' tabindex=-1 title='i18n_what_is_a_tracker' class='tooltip' href='https://www.eff.org/privacybadger#faq-What-is-a-third-party-tracker?'>dominio rastrador</a> por el momento. ¡Sigue navegando!",
+        "message": "Privacy Badger no ha detectado ningún <a target='_blank' tabindex=-1 title='i18n_what_is_a_tracker' class='tooltip' href='https://www.eff.org/privacybadger#faq-What-is-a-third-party-tracker?'>dominio rastrador</a> por el momento. ¡Siga navegando!",
         "description": "Shown on the Tracking Domains tab on the options page for new and recent installations."
     },
     "report_input_label": {
         "message": "Descripción",
         "description": ""
     },
     "non_tracker": {
-        "message": "Los siguiente dominos no parece que le estén rastreando",
+        "message": "No parece que los siguientes dominos le estén rastreando",
```

Comment:
Should this say "dominios"?

Reply:
@ghostwords, yes, I've fixed it. Thanks.